executor/partition_table.go

```diff
@@ -23,6 +23,9 @@ import (
 )
 
 func updateExecutorTableID(ctx context.Context, exec *tipb.Executor, recursive bool, partitionIDs []int64) error {
+	if exec == nil {
+		return nil
+	}
```

Comment:
we enter here and caused panic?

Reply:
Yes, `exec != nil` when `e.storeType == kv.TiFlash`.